pkg/restore/restore.go

```diff
@@ -1103,6 +1148,45 @@ func (ctx *context) restoreItem(obj *unstructured.Unstructured, groupResource sc
 	return warnings, errs
 }
 
+// renamePV returns a boolean indicating whether a persistent volume should be given a new name
+// before being restored, or an error if this cannot be determined. A persistent volume will be
+// given a new name if and only if (a) a PV with the original name already exists in-cluster, and
+// (b) in the backup, the PV is claimed by a PVC in a namespace that's being remapped during the
+// restore.
+func renamePV(ctx *context, obj *unstructured.Unstructured, client client.Dynamic) (bool, error) {
```

Comment:
small nitpick: we might want to call this `shouldRenamePV` to make it clearer that this call doesn't do the renaming